Source/Core/DolphinQt2/CheatsManager.h

```diff
@@ -0,0 +1,103 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <memory>
+#include <vector>
+
+#include <QDialog>
+
+#include "Common/CommonTypes.h"
+
+class QComboBox;
+class QDialogButtonBox;
+class QLineEdit;
+class QPushButton;
+class QRadioButton;
+class QSplitter;
+class QTabWidget;
+class QTableWidget;
+class QLabel;
+
+namespace UICommon
+{
+class GameFile;
+};
```

Comment:
Namespaces don't need semicolons to terminate them (this will cause -Wextra-semi warnings with clang). Ditto for the Core one below.